scripts/dev_setup.sh

```diff
@@ -27,18 +30,30 @@ function usage {
   echo "Installs or updates necessary dev tools for libra/libra."
   echo "-b batch mode, no user interactions and miminal output"
   echo "-p update ${HOME}/.profile"
-  echo "-o intall operations tooling as well: helm, terraform, hadolint, yamllint, vault, docker, kubectl, python3"
+  echo "-o install operations tooling as well: helm, terraform, hadolint, yamllint, vault, docker, kubectl, python3"
+  echo "-y install Move prover tools: z3, dotnet, boogie"
+  echo "-z installs or updates Move prover tools only"
```

Comment:
how about switching this out to only install the validator/rust tools if -v is passed in?   would be cleaner to be fully additive.

Reply:
I agree but leave that for another PR. I'm not sure whether anybody depends on the current semantics of the flags.

Reply:
only place depending on the flags is the tests in docker/ci/....

Reply:
Ok, done this now. There is a new option `-t` ("tools") for installing the build tools. I did a search over the project, and there where quite a few occurrences also in documentation. I hope we don't have any docs outside of the repo which refer to this.